mne/viz/_3d.py

```diff
@@ -51,7 +51,10 @@ def _fiducial_coords(points, coord_frame=None):
         points = (p for p in points if p['coord_frame'] == coord_frame)
     points_ = dict((p['ident'], p) for p in points if
                    p['kind'] == FIFF.FIFFV_POINT_CARDINAL)
-    return np.array([points_[i]['r'] for i in FIDUCIAL_ORDER])
+    if points_:
+        return np.array([points_[i]['r'] for i in FIDUCIAL_ORDER])
+    else:
+        return np.array([])
```

Comment:
For instance if you load a ctf dataset but don't supply a pos file, you get hpi dig points but no fids.

At some point it might be of interest to distinguish between digitized fiducials and hpi locaitons and MEG fit fids and hpi locations. For instance for my case i was constantly plotting digitized HPI locations along with the FIT hpi locations to visually see where errors might be cropping up.